qemu/tests/driver_in_use.py

```diff
@@ -65,7 +65,7 @@ def run_bg_stress_test(bg_stress_test):
                 {"sub_type": bg_stress_test})
             stress_thread.start()
         if not utils_misc.wait_for(lambda: check_bg_running(target_process),
-                                   120, 0, 1):
+                                   360, 0, 1):
```

Comment:
I just wanted to extend the checking time to make sure a valid check won't be missed.

In my original testing,I did not count Ds status as valid alive background test,which lead to  check_bg_running may fail if only 120s is provided.

Now the Ds stat is counted as well:list_cmd = "ps aux | grep rngd | grep [Rs,Ds]",120s should be ok,I will try more times to confirm.

Thanks.